tests/rules/no-subclass.ts

```diff
@@ -0,0 +1,180 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { stripIndent } from "common-tags";
+import rule = require("../../source/rules/no-subclass");
+import { ruleTester } from "../utils";
+
+ruleTester({ types: true }).run("no-subclass", rule, {
+  invalid: [
```

Comment:
This should include a valid snippet, too. See https://github.com/cartant/rxjs-tslint-rules/issues/87

Reply:
https://github.com/cartant/rxjs-tslint-rules/tree/master/test/v6/fixtures/issues/87

Reply:
I missed this one, adding it now.

Reply:
I'll go through the issue-related tests in `rxjs-tslint-rules` later. I've created a reminder issue. Might as well catch this one now, though. Thanks.